Tests/CreateReleaseNotes.Test.ps1

```diff
@@ -2,9 +2,6 @@
 Import-Module (Join-Path $PSScriptRoot '..\Actions\Github-Helper.psm1' -Resolve)
 Get-Module TestActionsHelper | Remove-Module -Force
 Import-Module (Join-Path $PSScriptRoot 'TestActionsHelper.psm1')
-Get-Module TelemetryHelper | Remove-Module -Force
-Import-Module (Join-Path $PSScriptRoot '..\Actions\TelemetryHelper.psm1')
-$errorActionPreference = "Stop"; $ProgressPreference = "SilentlyContinue"; Set-StrictMode -Version 2.0
 
```

Comment:
Shouldn't we set erroractionpreference - or os that happening elsewhere?

Reply:
We should :) Adding it back